maintainers/scripts/update-python-libraries

```diff
@@ -1,5 +1,3 @@
-#!/bin/sh
-build=`nix-build -E "with import (fetchTarball "channel:nixpkgs-unstable") {}; python3.withPackages(ps: with ps; [ packaging requests toolz ])"`
-python=${build}/bin/python
-exec ${python} pkgs/development/interpreters/python/update-python-libraries/update-python-libraries.py $@
-
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=channel:nixpkgs-unstable -i bash -p "python3.withPackages (ps: with ps; [ packaging requests ])" -p nix-prefetch-git
```

Comment:
Can we rely on nixpkgs from NIX_PATH? I don't have any channels on my systems so this would fail, right?

Reply:
Wby don't you give that command line a try and tell me?

Reply:
I wasn't on a PC when I wrote this but turns out it works.